### DISCUSSION TITLE
How do I implement/import this into my project?

### DISCUSSION THREAD

**Discussion by PsychoCoffee** [jrouwe/JoltPhysics#406] [Q&A]
Hello there,
I am pretty new to this (basically a noob at this), and while searching for a physics engine for my 3D project, I came across this and Nvidia's PhysX, but the issue is that I found no tutorials on implementing either one of them. This project still seems a lot easier and simpler for a starter project than PhysX though.

I did run the TestFramework application and test out some of the physics features and I am impressed.

1) Where do I find the folder with the include files for my project, and which ones are necessary for a simple physics project?

2) Right now I am only trying to get basic cube/cylinder collision done, so the player can jump and land on the floor, but may someone explain to me, or at least point me to the right direction of getting something like that done? I know its a giant subject to ask, but I am very lost lol.

Thanks in advance.

**Comment by jrouwe** [comment]
Hello,

Please take a look at the HelloWorld sample, this is the minimal example of how to integrate Jolt in your project.

Specific answers:

1. The include files are next to the cpp files, set your additional include folder to the JoltPhysics folder and include everything using <Jolt/...>.
2. Take a look at the CharacterVirtualTest demo, this includes basic player movement and has some basic shapes.
